scst-store/multicluster-setup.hbs.md

```diff
@@ -64,24 +64,13 @@ data:
 EOF
 ```
 
-### <a id='copy-amr'></a>Copy AMR CloudEvent Handler CA certificate from the View cluster
+### <a id='copy-amr'></a>Copy AMR CloudEvent Handler CA certificate data from the View cluster
 
 With your kubectl targeted at the View cluster, you can view AMR CloudEvent Handler's TLS
 CA certificate. To copy the CA certificate into a `ceh_ca.yaml` file:
 
 ```console
-CEH_CA_CERT=$(kubectl get secret -n amr-observer-system amr-cloudevent-handler-ingress-cert -o json | jq -r ".data.\"ca.crt\"")
-cat <<EOF > ceh_ca.yaml
----
-apiVersion: v1
-kind: Secret
-type: Opaque
-metadata:
-  name: truststore
-  namespace: amr-observer-system
-data:
-  ca.crt: $CEH_CA_CERT
-EOF
+CEH_CA_CERT_DATA=$(echo $(kubectl get secret -n metadata-store amr-cloudevent-handler-ingress-cert -o json | jq -r ".data.\"ca.crt\"") | base64 -d)
```

Comment:
Do you need the `echo`?  Wouldn't this work?
```
CEH_CA_CERT_DATA=$(kubectl get secret -n metadata-store amr-cloudevent-handler-ingress-cert -o json | jq -r ".data.\"ca.crt\"" | base64 -d)
```

Reply:
That works